### ISSUE TITLE
Data Request

### ISSUE THREAD

**Issue by SweetFov** [liuzqt/keyword_spotting#3]
Hey, I am pretty interested in this project, and currently I'm reading your codes. 
Could you provide the data in your demo, so i can run your demo and figure out how you built this demo? I would be very thankful if you do this!
My e-mail : sweetfov@126.com

**Comment by SweetFov** [comment]
I am still reading your project, and I am understanding your dataset. It seems that each of your wav file, have been labelled their corresponding keyword label, and their start/end time?  And could you share your model's accuracy?

**Comment by liuzqt** [comment]
@SweetFov Sorry the data was from my company's database and it's not supposed to exposed. At the beginning I'm trying to use cross-entropy like method, i.e. frame-wise training, that's why I used pre-segmented and aligned data label. (The alignment and label for start/end time is done by our speech recognition model) 
However, the model can only reach about 85% accuracy, and actually I suspect that it highly rely on the data quality(the segment and alignment accuracy). Even our speech model can not perform very well in segmenting the data.   So now I shifted to rnn+ctc structrue, where the label is only text, so that I can make sure my label is golden, and I can add as more data as I want. The new model reach about 95% accuracy.

**Comment by SweetFov** [comment]
@liuzqt  So your new model(rnn+ctc）, is a general LVSCR system(transform speech to corresonding text regardless of the keywords), or partically designed for keyword spotting?

**Comment by liuzqt** [comment]
@SweetFov all irrelevant words are maped to one class: garbage word, so it not a balanced output space mapping. The trainning data are as large as LVSCR system in order to provide adequate negative sample.   Of course I oversample the positive samples, otherwise they are too sparse.

**Comment by SweetFov** [comment]
So as far as I'm concerned, you got adequate speech data with corresponding transcripts; Then, you just label all the irrelevant words to one class (garbage word), and send those 'relabeled ' data to your KWS model for training? Are your codes available in your Github? That's truly a brand new way to do KWS!

**Comment by SweetFov** [comment]
BTW, how do you oversample the positive samples?

**Comment by SweetFov** [comment]
@liuzqt  Oh I got your RNN+CTC github. Exciting!

**Comment by liuzqt** [comment]
@SweetFov the oversampling rate depends on your keyword. In my case, the positive sample(include the whole keyword or part of the keyword) take about 1/3 of the whole training data.
